crates/telio-dns/src/forward.rs

```diff
@@ -218,6 +218,10 @@ impl Authority for ForwardAuthority {
         let name: LowerName = name.clone();
         let resolve = self.resolver.lookup(name, rtype).await;
 
+        if resolve.is_err() {
+            telio_log_warn!("DNS name resolution failed with {:?}", resolve);
```

Comment:
will this contain server + error message?

Reply:
This is an example of the error message, when routing table does not have route towards the DNS server:
```
2024-06-28T08:49:33.384834Z  WARN telio_dns::forward: 222: DNS name resolution failed with Err(ResolveError { kind: Proto(ProtoError { kind: Io(Os { code: 113, kind: HostUnreachable, message: "No route to host" }) }) })
```

Reply:
Upstream servers are configured over FFI layer, where it may be seen.